.github/workflows/ci_cd.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.9', '3.10', '3.11', '3.12']
```

Comment:
And 3.9 should be dropped as well, right?

Reply:
Completely right! Let me fix it